core/src/client/importer.rs

```diff
@@ -263,10 +263,9 @@ impl Importer {
 
         // Enact Verified Block
         let db = client.state_db().read().clone(&parent.state_root());
-        let coordinator = &self.coordinator;
+        let validator = self.validator.borrow();
```

Comment:
```rust
let validator = &*self.validator;
```
It' the same code as yours. You don't need to use `Borrow` unless you want a generic code.

Reply:
Ok, thanks. I fixed both `miner` and `importer`